tests/robot-tests/tests/visual_testing/tables_and_charts.robot

```diff
@@ -0,0 +1,163 @@
+${SECONDARY_STATS_FOLDER}=      secondary-stats
+${CONTENT_SECTIONS_FOLDER}=     content-sections
+${FAST_TRACKS_FOLDER}=          fast-tracks
+
+*** Keywords ***
+Check release
+    [Arguments]    ${release}
+
+    Log to console    \n\n\t=====================================================================\n
+    Log to console    \tProcessing release at %{PUBLIC_URL}${release.url}
+
+    user navigates to public frontend    %{PUBLIC_URL}${release.url}
+
+    IF    ${release.has_key_stat_blocks} is ${TRUE}
+
+        Log to console    \n\tCapturing Key Stats:
+
+        user waits until page contains element    xpath://h2[contains(text(), "Headline facts and figures")]
+        user scrolls to element    xpath://h2[contains(text(), "Headline facts and figures")]
+        user waits until page contains element    id:releaseHeadlines-summary
+
+        FOR    ${content_block}    IN    @{release.key_stat_blocks}
+            check key stats table    ${content_block}
+        END
+    END
+
+    IF    ${release.has_secondary_stat_blocks} is ${TRUE}
+        Log to console    \n\tCapturing Secondary Stats:
+
+        FOR    ${content_block}    IN    @{release.secondary_stat_blocks}
+            check secondary stats table    ${content_block}
+        END
+    END
+
+    IF    ${release.has_content_section_blocks} is ${TRUE}
+        Log to console    \n\tCapturing Content Section Data Blocks:
+
+        FOR    ${content_block}    IN    @{release.content_section_blocks}
+            IF    ${content_block.has_table_config} is ${TRUE}
+                check content block table    ${content_block}
+            END
+        END
+    END
+
+    IF    ${release.has_fast_track_blocks} is ${TRUE}
+        Log to console    \n\tCapturing Fast Tracks:
+
+        FOR    ${content_block}    IN    @{release.fast_track_blocks}
+            check fast track table    ${content_block}
+        END
+    END
+
+Check Fast Track Table
+    [Arguments]    ${content_block}
+    user navigates to public frontend    %{PUBLIC_URL}${content_block.content_url}
+    user waits until page contains element    id:tableToolWizard
+    ${filepath}=    user takes screenshot of element    id:tableToolWizard
+    ...    ${SNAPSHOT_FOLDER}/${content_block.release_id}/${FAST_TRACKS_FOLDER}/${content_block.content_block_id}-table.png
+    user takes html snapshot of element    id:tableToolWizard
+    ...    ${SNAPSHOT_FOLDER}/${content_block.release_id}/${FAST_TRACKS_FOLDER}/${content_block.content_block_id}-table.html
+    log content block details    ${content_block}    Fast Track    ${filepath}
+
+Check Content Block Table
+    [Arguments]    ${content_block}
+    user waits until page contains element    id:content-${content_block.content_section_position}
+    ${accordion}=    user opens accordion section with id    content-${content_block.content_section_position}
+    ...    id:content
+    user scrolls to element    ${accordion}
+    user scrolls to the top of the page
```

Comment:
Why are we scrolling up & down multiple times here? 

```
    user scrolls to the top of the page
    user scrolls to the bottom of the page
    user scrolls to the top of the page
    user scrolls to the bottom of the page
```

Reply:
Yeah it was a bit strange, and I agree this is odd to have in!  This was time pressure, but effectively if the thing you opened was above where you're currently looking, you need to scroll up to force lazy load.  If below, you need to scroll down to force lazy load.  Because we don't know if it's above or below where we're currently looking, we have to at least go all the way one way, then all the way the other to ensure we pass it and get it to lazy load.  We can't scroll to it cos it's not yet initialised content.

At this point it was against the clock so need something that just worked.  Happy to revisit though to find something less hacky when we've got time for sure, if that's OK with you?